src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ClusterBox.tsx

```diff
@@ -0,0 +1,86 @@
+// Libraries
+import React, {FC, useCallback, useContext, useMemo} from 'react'
+import classnames from 'classnames'
+import {FlexBox, FlexDirection} from '@influxdata/clockface'
+
+// Style
+import './ClusterBox.scss'
+
+// Components
+import {CreateOrgContext, ProviderID} from './CreateOrganizationContext'
+import {Cluster} from 'src/client/unityRoutes'
+import {AWSLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AWSLogo'
+import {AzureLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AzureLogo'
+import {GCPLogo} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/GCPLogo'
+import {AzureLogoWithText} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AzureLogoWithText'
+import {GCPLogoWithText} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/GCPLogoWithText'
+import {AWSLogoColor} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderLogos/AWSLogoColor'
+
+const LOGOS = {
+  AWS: (showLogoWithText: Boolean) =>
```

Comment:
I don't think this object needs to be allcaps

Reply:
i planned on using `BillingProvider` as type for keys. But there was a lot of `toLowerCase` and back and forth in typecasting. So, I kept it the way we receive `providerId` in response.